pycaption/webvtt.py

```diff
@@ -237,12 +247,22 @@ def _write_caption(self, caption):
 
         output = u''
 
+        cue_style_tags = [u'', u'']
+        if u'class' in caption.style:
+            style_class = caption.style['class']
+            style = caption_set.get_style(style_class)
+            for key, value in style.iteritems():
```

Comment:
We're planning to make pycaption compatible with python3. 
Because of this, please use `.items()`. The styles dict isn't big, anyway, so this shouldn't be a problem in python2 either.